docker-compose.yml

```diff
@@ -17,43 +16,31 @@ services:
       - P2P_ENVIRONMENT=${P2P_ENVIRONMENT}
       - P2P_DISCOVERY_NAMESPACE=${P2P_DISCOVERY_NAMESPACE}
     ports:
-      - "8080:8080"
-    networks:
-      - app-network
-    command: ["./observer", "start", "--rpc-url", "${RPC_URL}", "--contract-address", "${CONTRACT_ADDRESS}", "--p2pkey", "${P2P_KEY}"]
+      - "23003:23003"
+    command: ["start", "--rpc-url", "${RPC_URL}", "--contract-address", "${CONTRACT_ADDRESS}", "--p2pkey", "${P2P_KEY}"]
 
   db:
     image: postgres:14.12
-    container_name: postgres-db
     environment:
       - POSTGRES_USER=postgres
       - POSTGRES_PASSWORD=${DB_PASSWORD}
       - POSTGRES_DB=shutter_metrics
     volumes:
       - pgdata:/var/lib/postgresql/data
-    ports:
-      - "5432:5432"
-    networks:
-      - app-network
     healthcheck:
       test: ["CMD-SHELL", "pg_isready -U postgres"]
-      interval: 10s
-      timeout: 5s
-      retries: 5
+      start_interval: "2s"
+      start_period: "30s"
```

Comment:
This decreases the time required until the container is considered healthy without running the healthcheck unnecessarily often after the service is up.